async_simple/Executor.h

```diff
@@ -119,7 +121,7 @@ class Executor {
 
     // IOExecutor accepts IO read/write requests.
     // Return nullptr if the exeuctor doesn't offer an IOExecutor.
-    virtual IOExecutor *getIOExecutor() = 0;
+    virtual IOExecutor *getIOExecutor() { return nullptr; }
```

Comment:
I think this one should throw error too.

Reply:
ok